MonoGame.Framework/Windows/WinFormsGameForm.cs

```diff
@@ -137,5 +153,15 @@ protected override void WndProc(ref Message m)
 
             base.WndProc(ref m);
         }
+
+        private static ushort HiWord(IntPtr word)
+        {
+            return (ushort)(((ulong)word >> 16) & 0xffff);
+        }
+
+        private static short GetWheelDeltaWParam(IntPtr wParam)
+        {
+            return (short)HiWord(wParam);
```

Comment:
This is another place where I was trying to match common macros that people might try to search for online.  I can remove this function and inline the call though.